jira_automation/src/main/java/io/dockstore/jira/Utils.java

```diff
@@ -45,4 +133,44 @@ public static Optional<String> findJiraIssueInBody(GHIssue ghIssue) {
         return Optional.empty();
     }
 
+    private static Builder authorizedRequestBuilder() {
+        return HttpRequest.newBuilder().header("Authorization", getAuthHeaderValue());
+    }
+
+    private static String getAuthHeaderValue() {
+        final String username = System.getenv(JIRA_USERNAME);
+        final String jiraToken = System.getenv(JIRA_TOKEN);
+        return "Basic %s".formatted(
+                Base64.getEncoder().encodeToString((username + ':' + jiraToken).getBytes()));
```

Comment:
To be sure you're converting to UTF-8, use `getBytes(charset)`, otherwise the platform's default charset will be used.